core/src/main/java/org/elasticsearch/client/Client.java

```diff
@@ -420,36 +414,6 @@
     MultiTermVectorsRequestBuilder prepareMultiTermVectors();
 
     /**
-     * Percolates a request returning the matches documents.
-     */
-    ActionFuture<PercolateResponse> percolate(PercolateRequest request);
```

Comment:
One thing that has come up with reindex being in a plugin: users of the Java API have trouble finding it. I don't know what, if anything, we should do about that. This will have the same problem.